src/network/addrthread.py

```diff
@@ -7,7 +7,7 @@
 # magic imports!
 import connectionpool
 from protocol import assembleAddrMessage
-from queues import addrQueue  # FIXME: init with queue
```

Comment:
Please check Fixme comment, i'm not sure whether i remove this or not.

Reply:
I'm not sure either, let's leave it as it is.